### PR TITLE
add: Free Programming Books to projects open source

### LINKED ISSUE #1997: [ADD] free-programming-books of open_source/projects
### Category Name

open_source

### Subcategory Name

projects

### Additional Context

_No response_

### Checklist

- [X] I have checked the existing [issues](https://github.com/rupali-codes/LinksHub/issues?q=is%3Aissue+)
- [X] I have read the [Contributing Guidelines](https://github.com/rupali-codes/LinksHub/blob/main/CONTRIBUTING.md)
- [X] I have checked that this category does not exist
- [X] I have checked that this subcategory does not exist
- [X] I am willing to work on this issue (optional)
- [X] I am participating in Hacktoberfest 2023

### PR DESCRIPTION
**Pull request by codeOraaange**
## Fixes Issue

Closes #1997 

## Changes proposed

Add Free Programming Books Github Repo.

### CODE REVIEW

File: database/open_source/projects.json [hunk 1]
```diff
@@ -152,6 +152,13 @@
     "url": "https://github.com/TheTorProject",
     "category": "open-source",
     "subcategory": "projects"
+  },
+  {
+    "name": "Free Programming Books",
+    "description": "Freely available programming books.",
```
Comment (CBID2) [inline database/open_source/projects.json:158]: [suggestion] This description could use a bit more detail. Consider mentioning the purpose of this project and why people should contribute to it. That way, people would become more interested in contributing.
```suggestion
    "description": "Freely available programming books.",
```

**Review by CBID2 — CHANGES_REQUESTED** [review]
Hi @codeOraaange, your PR is off to a great start. I just gave some suggestions on how to improve your description.

**Comment by codeOraaange** [comment]
Thanks for suggestion @CBID2, I've adjusted.

**Review by CBID2 — APPROVED** [review]
Looks great

**Pull request merged**